src/s2wasm.h

```diff
@@ -1193,6 +1198,42 @@ class S2WasmBuilder {
         }
       }
     }
+    if (!!startFunction) {
```

Comment:
But we use Name extensively, for performance reasons. I'd rather use Name for startFunction as well. Is there a reason not to?